tests/unit/email-dropdown.spec.js

```diff
@@ -112,6 +112,21 @@ describe("EmailDropdown.vue", () => {
       expect(wrapper.emitted().input[1]).to.have.length(1);
       expect(wrapper.emitted().input[1][0]).to.be.equal("hello@gmail.");
     });
+
+    it("returns 'true' if input is empty after clicking clean (X) button", async () => {
+      propsData.domains = ["gmail.com", "google.com"];
```

Comment:
Since domains `["google.com"]` is by default you don't need more than one here in order to make this test pass.
```suggestion
```